### ISSUE TITLE
Toolbar not working in DNN9..5.0

### ISSUE THREAD

**Issue by improwise** [dnnsoftware/Dnn.Platform#3612]
Seems like the DNNToolbar  (the one on the bottom where you add new modules etc) is not working as it should after upgrading to DNN 9.5.0. Instead, you seem to be in edit mode as soon as you login (which seem to be working) but you can't add new modules etc. Clicking the pen changes from grey to blue and back, but toolbar is not present. 

We notice that in the console we see errors about DNN trying to reach dnn.controls.dnntoolbar.js in root, when it is infact in /js. Manually copying the file to root will make that error go away, but the actual problem seem to remain. 

Have tried several portals and themes so I don't think its a theme/skin problem. 

So, in summary, we are stuck in some kind of semi-edit mode, in which we can make some changes but not all, and we also can not seem to exit into view mode.

**Comment by improwise** [comment]
Perhaps I should add that this is a DNN site upgraded from DNN 9.2

**Comment by improwise** [comment]
I am wondering if we are perhaps doing something wrong here, as before, a click on the pencil button would bring up the toolbar or hide it, but now it seems to change between locked edit mode and not locked and for obvious reasons, a click on the same icon can not do two different things. 

Is there a usability change here that we have missed and if so, how do you get to the normal edit mode where you can add modules etc?

**Comment by improwise** [comment]
This seem to be related:

https://www.dnnsoftware.com/answers/edit-mode-not-working-after-upgrade-to-910

**Comment by valadas** [comment]
I cannot reproduce the issue, can you try making a new page with the default theme and no modules on it to see if that issue still happens on it.

**Comment by improwise** [comment]
Turns out this was a user error (mine to be precise). When trying to clean up the web.config file after some failed uninstallations of DNN modules (like the security patch 2017) we managed to comment out this section as well:

      <add name="EditBarModule" type="Dnn.EditBar.UI.HttpModules.EditBarModule, Dnn.EditBar.UI" preCondition="managedHandler" />
      <add name="PersonaBarModule" type="Dnn.PersonaBar.UI.HttpModules.PersonaBarModule, Dnn.PersonaBar.UI" preCondition="managedHandler" />
